src/test/java/com/karankumar/bookproject/ui/statistics/StatisticsViewTest.java

```diff
@@ -133,6 +134,18 @@ void withoutGenreInformationOtherStatisticsStillShown() {
         pageStatisticsArePresent();
     }
 
+    @Test
+    void withOnlyOneBookTheMostLikedAndLeastLikedBookShouldBeAbsent() {
+        // given
+        populateDataWithOnlyOneBook(bookService, predefinedShelfService);
+
+        // when
+        statisticsView = new StatisticsView(predefinedShelfService);
+
+        // then
+        likedBookStatisticsAreAbsent();
```

Comment:
Could you please replace this with `ratingStatisticsAreAbsent()`? I know this includes the average rating, but I don't think we should show that either if there's only one book.

Sorry to increase the scope, but I think it's easier to add this now while you're working on this. It only just occurred to me after seeing this test.

Reply:
Done with 28fb504ce63949b3c1cb19dadcd8e89283aaa2a3